Assets/HoloToolkit/Utilities/Scripts/Billboard.cs

```diff
@@ -53,7 +53,7 @@ private void OnEnable()
         {
             if (TargetTransform == null)
             {
-                TargetTransform = CameraCache.Main.transform;
+                TargetTransform = CameraCache.Main?.transform;
```

Comment:
I'm not sure ?. is available with the legacy C# compiler (supporting .net 3.5). please change this to a traditional null check (in your Update() change as well).

thx!